compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/text/LongPressTextDragObserver.kt

```diff
@@ -86,8 +86,7 @@ internal suspend fun PointerInputScope.detectDownAndDragGesturesWithObserver(
         launch(start = CoroutineStart.UNDISPATCHED) {
             detectDragGesturesWithObserver(observer)
         }.invokeOnCompletion {
-            // Otherwise observer won't be notified if
-            // composable was disposed before the drag cancellation
+            // b/288931376: detectDragGestures do not call onDragCancel when composable is disposed.
```

Comment:
If it isn't cherry-pick from AOSP, let's revert it here. It will help avoiding merge conflicts.

Reply:
It was my comment from #1000